agent/util-scripts/tool-meister/pbench-tool-data-sink

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python3
+
+"""Simple command-line wrapper to keep the tool data sink from being in the
+CLI command set, while still allowing it to be invoked by the container entry
+point.
+"""
```

Comment:
> I don't see how the existence of this script prevents the TDS from being in the CLI command set

When the pbench-agent RPM is installed, `/opt/pbench-agent/util-scripts/tool-meister` will not be in the `PATH`.

> nor do I see how this script is necessary to allow the tool to be invoked by the container entry point.

Not sure there is much value collapsing all these files.  The `pbench-tool-data-sink` and `pbench-tool-meister` executables are the Python3 shell wrappers to invoke the respective main classes.  The `*-ep` are the specific entry points behaviors for each of the containers, and the `pbench-tool-meister-remote` is what is invoked when running Tool Meisters via SSH.

All of these are separate concerns.

Reply:
> When the `pbench-agent` RPM is installed, `/opt/pbench-agent/util-scripts/tool-meister` will not be in the PATH.

Wouldn't that be just as true if we didn't create this file?  (That is, if you really want to create this file, perhaps the header comment should say something different:  pointing out that this script is used as the container entry point is good; mentioning that it is not and should not be a part of the CLI command set is good; but, that part about how this script prevents that from happening is confusing, at least to me.)

> Not sure there is much value collapsing all these files.

The value is in reducing the number of moving parts.  The invocation of the TDS is so straightforward that it really doesn't need a script wrapped around it (it really is just a Python command, AFAICS).  Omitting the script means one fewer file to be included in the RPM, installed in the right place, and otherwise tracked and debugged.  If the script actually provided some functionality (or any other significant positive feature) I would say go with it, but it's just a forwarding wrapper, and we don't need it here (the conclusions for the other scripts might be different, but I doubt it).